cmd/aro/mirror.go

```diff
@@ -44,6 +43,13 @@ func mirror(ctx context.Context, log *logrus.Entry) error {
 		}
 	}
 
+	environment, err := azureenvironment.AzureEnvFromEnvVar()
```

Comment:
I think that mirror should consume an env.Core rather than doing this

Reply:
done